OmiseSDK/Omise3DSViewController.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+import WebKit
+
+
+/// Delegate to receive 3DS verification events.
+public protocol Omise3DSViewControllerDelegate: class {
+    
+    func omise3DSViewController(_ viewController: Omise3DSViewController, didFinish3DSProcessWithRedirectedURL redirectedURL: URL?)
```

Comment:
I think `didComplete` might fits better in this case. `Finish` can imply a failing finish.